packages/passport/sdk/src/config/config.ts

```diff
@@ -96,6 +96,7 @@ export class PassportConfiguration {
         }),
         passport: createConfig({
           basePath: overrides.passportMrBasePath,
+          headers: overrides.headers,
```

Comment:
yeah, i am not convinced that we should add this to fix a sample app issue. as we are exposing headers to other clients, might have risks but meantime i don't know if the headers will be used by other clients.